Point to trunk instead of develop

hypest opened the pull request:
### Fix #971 

Updates the README so the links towads `develop` now point to `trunk`, since `develop` has been removed in favor of `trunk`.

### Test
Check that the README now shows the 2 images correctly.

👋 @planarvoid , added you as a reviewer since I know your are active on this repo these days, thanks!

Make sure strings will be translated:

- [x] If there are new strings that have to be translated, I have added them to the client's `strings.xml` as a part of the integration PR.

hypest committed:
Point to trunk instead of develop

planarvoid reviewed (changes requested):
I've tested the links and all work except the one pointing to the iOS repo (which doesn't seem to be moved to `trunk` yet (link - https://github.com/wordpress-mobile/AztecEditor-iOS/raw/trunk/RepoAssets/aztec.png)

hypest committed:
Add Aztec repo image asset and point to it

hypest commented:
Oh, good catch @planarvoid ! I went ahead and added the asset to this repo too, and adjusted the pointer as well. Let me know what you think.

The new pointer won't work yet in this PR, but it should after merging it to trunk.

planarvoid reviewed (approved):
looks good! thanks for the changes 🙇
